src/app/operators/_operator-theme.scss

```diff
@@ -2,7 +2,7 @@
 
 $operator-border: #f3f3f3;
 $operator-active: #62757f;
-$link-color: #2196f3;
+$link-color:#0a6fc2;
```

Comment:
Not a big deal, but isn't better to keep the space between colon and #? Just a cosmetic point